schemas/com.sendgrid/bounce/jsonschema/2-0-0

```diff
@@ -0,0 +1,107 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a SendGrid bounce event",
+  "self": {
+    "vendor": "com.sendgrid",
+    "name": "bounce",
+    "version": "2-0-0",
+    "format": "jsonschema"
+  },
+  "type": "object",
+  "properties": {
+    "ip": {
+      "description": "the IP address used to send the email. For open and click events, it is the IP address of the recipient who engaged with the email.",
+      "type": "string",
+      "format": "ipv4"
+    },
+    "timestamp": {
+      "description": "the UNIX timestamp of when the message was sent",
```

Comment:
I think "UNIX timestamp" assumes its an seconds since the epoch, but `date-time` assumes its a ISO8601 string.

Reply:
"UNIX timestamp" comes from the Sendgrid documentation. But as you noted elsewhere, we're schemaing the *output* of the webhook and not the *input*, so the proper format is `date-time`. I have updated the descriptions of the `timestamp` field across all schemas.